### PR TITLE
Updates to build with Windows toolchain

### PR DESCRIPTION
**Pull request by shruthi-ravi**
###Testing

Ran `make html` without errors. Verified the generated htmls look ok. I don't know how to verify the image looks correct in the html.

### CODE REVIEW

File: docs/source/cross_compile/config_vs_code.rst [hunk 1]
```diff
@@ -41,26 +41,31 @@ Visual Studio Code, follow the steps below.
...
    Documents>\\NI Linux Real-Time Template\\*.
 
-2. Launch VS Code.
+2. In a new command prompt, navigate to the toolchain folder and run the
+   environment-setup-core2-64-nilrt-linux.bat script.
 
-3. | In VS Code, select **Open folder…** from the landing page or **File
+3. Launch VS Code from the same command prompt, to ensure that the toolchain
+   environment is set up correctly. Hereafter, always launch VS Code for
+   cross-compilation after setting up the environment.
```
Comment (SparkingSpork) [inline docs/source/cross_compile/config_vs_code.rst:49]: Is this always necessary even with the config set up in the VS Code environment? I thought this script just added things to the PATH/environment variables. Does it do more than that now?

**Review by SparkingSpork — APPROVED** [review]
Looks good to me. Before pulling, I'm curious if the bash script is necessary or potentially removes the need for some of the configuration elsewhere in the tutorial.

File: docs/source/cross_compile/config_vs_code.rst (hunk 1 above)
Comment (shruthi-ravi) [reply docs/source/cross_compile/config_vs_code.rst:49]: Frankly, I did not try setting up any other way due to instructions [here](https://ni.visualstudio.com/DevCentral/_boards/board/t/RTOS/Work%20Items/?workitem=2105299). The script still does only that, but it seems to add several compiler command-line arguments that is missing from the the VS Code config.

Comment (SparkingSpork) [reply docs/source/cross_compile/config_vs_code.rst:49]: I can see a future state where we update the tutorials to run that setup as part of the build process, but for now I think this is fine.

**Pull request merged**